packages/devtools_app/lib/src/memory/memory_screen.dart

```diff
@@ -282,6 +286,15 @@ class MemoryBodyState extends State<MemoryBody>
       }
     });
 
+    addAutoDisposeListener(controller.androidCollectionEnabled, () {
+      setState(() {
```

Comment:
can we add a local var `androidCollectionEnabled` that tracks the value of `controller.androidCollectionEnabled` and is updated in `setState()`? then we can use that local var below.

Reply:
Done.

Reply:
Done.